### PR TITLE
eCR Refiner stubbed service

### LINKED ISSUE #1554: Create a new stubbed out eCR minifier building block
Create the directory for the eCR minifier and the implement the standard status check endpoint.

### PR DESCRIPTION
**Pull request by m-goggins**
# PULL REQUEST

## Summary
Stubs out the service for the eCR Refiner work.

## Related Issue
Fixes #1554 and #1559

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/refiner/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+../dibbs
+pathlib
```
Comment (bamader) [inline containers/refiner/requirements.txt:2]: Not a problem with this file or PR, but might be good to start up a `dev-requirements.txt` file out of the gate that we can put `pytest` in

**Review by bamader — APPROVED** [review]
Looks right to me!

**Review by robertandremitchell — APPROVED** [review]
looks good

File: containers/refiner/requirements.txt (hunk 1 above)
Comment (m-goggins) [reply containers/refiner/requirements.txt:2]: Good call! I'll add one

**Pull request merged**